lib/slack-ruby-bot-server/config.rb

```diff
@@ -3,11 +3,24 @@ module Config
     extend self
 
     attr_accessor :server_class
+    attr_accessor :database
```

Comment:
Probably `database_type` is more appropriate. Even better, since we're talking about the adapter to access a database, maybe even `database_driver`?